File: eMovieFinder/eMovieFinder.Model/Dtos/Requests/MovieActor/MovieActorUpdateRequest.cs
namespace eMovieFinder.Model.Dtos.Requests.MovieActor
{
    public class MovieActorUpdateRequest
    {
        public int? MovieId { get; set; }
        public int? ActorId { get; set; }
        public string? CharacterName { get; set; }
    }
}

File: eMovieFinder-UI/emovie_finder_desktop/lib/src/models/dtos/requests/movie/movie_actor_update_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'movie_actor_update_request.g.dart';

@JsonSerializable()
class MovieActorUpdateRequest {
  final int? movieId;
  final int? actorId;
  final String? characterName;

  MovieActorUpdateRequest({
    this.movieId,
    this.actorId,
    this.characterName
  });

  factory MovieActorUpdateRequest.fromJson(Map<String, dynamic> json) => _$MovieActorUpdateRequestFromJson(json);
  Map<String, dynamic> toJson() => _$MovieActorUpdateRequestToJson(this);
}